packages/dynamic_layouts/README.md

```diff
@@ -13,25 +13,138 @@ and the Flutter guide for
 [developing packages and plugins](https://flutter.dev/developing-packages).
 -->
 
-TODO: Put a short description of the package here that helps potential users
-know whether this package might be useful for them.
+<!-- TODO(DavBot02): Put a short description of the package here that helps potential users
+know whether this package might be useful for them.-->
 
 ## Features
+This package provides support for multi sized tiles and different layouts.
+Currently the layouts that are implemented in this package are `Stagger` and
+`Wrap`.
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+The following are some demos of how each of the grids look.
+
+A stagger grid demo:
+
+<!-- TODO(snat-s): Add stagger video demo -->
+
+A wrap demo:
+
+<!-- TODO(snat-s): Add wrap video demo -->
+
+### Stagger Features
+
+`DynamicGridView` is a subclass of `GridView` and gives access
+to the `SliverGridDelegate`s that are already implemented in the Flutter
+Framework. Some `SliverGridDelegate`s are `SliverGridDelegateWithMaxCrossAxisExtent` and
+`SliverGridDelegateWithFixedCrossAxisCount`. This layout can be used with
+`DynamicGridView.stagger`.
+
+### Wrap Features
+
+The Wrap layout is able to do runs of different widgets and adapt accordingly with
+the sizes of the children. It can leave spacing with `mainAxisSpacing` and
+`crossAxisSpacing`.
+
+Having different sizes in only one of the axis is possible by
+changing the values of `childCrossAxisExtent` and `childMainAxisExtent`. These
+values by default are ignored, but changing `childCrossAxisExtent` to be
```

Comment:
The sentence about the values being ignored is confusing. What do I need to do to un-ignore the values? What extents are used while the value is ignored?

Reply:
Yeah, I think users will likely better understand loose constraints versus tight constraints. By default they are loose, allowing the child to be whatever finite size it wants, if any of these extents are set by the user, they will be tight constraints for that axis, forcing that dimension of the child's size.